projects/austria/.env

```diff
@@ -0,0 +1,3 @@
+COMPOSE_PROJECT_NAME=pelias
+DATA_DIR=./data
+DOCKER_USER=0
```

Comment:
We will shortly merge https://github.com/pelias/docker/pull/220 which will prevent users running as root.

This value should either be set as the default for other projects (1000) or removed if merged after #220

Reply:
Thank you very much for your quick review. As requested I have updated the docker_user to 1000.

Reply:
Hey, sorry to be a pain but could you please remove that line 🙏 
We merged https://github.com/pelias/docker/pull/220 already and I've removed it from the other projects already.

Reply:
oh sorry misunderstanding. Of course. As requested I have removed the line.